### PR TITLE
Misc housekeeping tasks / issue resolutions

### LINKED ISSUE #152: Contradicting docker-compose.yml examples.
## Overview

- What's your problem about?
Contradicting ```docker-compose.yml``` displayed on Quickstart page, and in ```example``` folder download. Which one is the current example?
- What is your operating system?
N/A
- What is your docker version? `docker version`
N/A

## `docker-compose.yml`
In the folder named ```example``` downloaded from the [Quickstart page](https://github.com/visiblevc/wordpress-starter/tree/master/example#quickstart) and identical (of course) to the one in the repo. Download link shown right below the line _"Download this example to your current working directory"_.

```yml
version: "3"
services:
    wordpress:
        image: visiblevc/wordpress

        # required for mounting bindfs
        cap_add:
            - SYS_ADMIN
        devices:
            - /dev/fuse
        # required on certain cloud hosts
        security_opt:
            - apparmor:unconfined

        ports:
            - 8080:80
            - 443:443
        volumes:
            - ./data:/data
            - ./scripts:/docker-entrypoint-initwp.d
        environment:
            DB_NAME: wordpress
            DB_PASS: root
            PLUGINS: >-
                academic-bloggers-toolkit
                co-authors-plus
                [WP-API]https://github.com/WP-API/WP-API/archive/master.zip

    db:
        image: mariadb:10 # or mysql:5.7
        volumes:
            - data:/var/lib/mysql
        environment:
            MYSQL_ROOT_PASSWORD: root

    phpmyadmin:
        image: phpmyadmin/phpmyadmin
        ports:
            - 22222:80

volumes:
    data:
```

## `docker-compose.yml`
Shown on the same [Quickstart page](https://github.com/visiblevc/wordpress-starter/tree/master/example#quickstart) below the line _"The only thing you need to get started is a docker-compose.yml file:"_.

```yml
version: '3'
services:
  wordpress:
    image: visiblevc/wordpress:latest
    ports:
      - 8080:80
      - 443:443
    volumes:
      - ./data:/data # Required if importing an existing database
      - ./tweaks.ini:/usr/local/etc/php/conf.d/tweaks.ini # Optional tweaks to the php.ini config
      - ./wp-content/uploads:/app/wp-content/uploads
      - ./yourplugin:/app/wp-content/plugins/yourplugin # Plugin development
      - ./yourtheme:/app/wp-content/themes/yourtheme   # Theme development
    environment:
      DB_HOST: db # must match db service name below
      DB_NAME: wordpress
      DB_PASS: root # must match below
      PLUGINS: >-
        academic-bloggers-toolkit,
        co-authors-plus,
        [WP-API]https://github.com/WP-API/WP-API/archive/master.zip,
      URL_REPLACE: localhost:8080
      WP_DEBUG: 'true'
  db:
    image: mysql:5.7 # or mariadb:10
    volumes:
      - data:/var/lib/mysql
    environment:
      MYSQL_ROOT_PASSWORD: root
volumes:
  data: {}
```

## Project structure

```
N/A
```

## `docker-compose up` output

```
N/A
```

### LINKED ISSUE #160: Max upload file size of 2mb!
The first thing I do in wordpress is to upload a file but this image only has maximum file size upload for wordpress set to 2mb! Do I need to mount the config to fix it?

### PR DESCRIPTION
**Pull request by dsifford**
This PR isn't very exciting. It does the following:

1. bumps the default theme to the newly available `twentytwenty` theme.
2. includes the `must-use` plugins directory in the search for plugin volumes.
3. bumps the max upload filesize to 50M ([context here](https://github.com/visiblevc/wordpress-starter/issues/160#issuecomment-544561961))
4. runs shfmt on `run.sh`
5. Address contradicting examples.
6. Improve plugin volumes check to also include plugins that are a single php file

Closes #160
Closes #152 

Ping @karellm

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -42,7 +42,6 @@ RUN echo "deb http://ftp.debian.org/debian $(sed -n 's/^VERSION=.*(\(.*\)).*/\1/
         zip \
     && docker-php-ext-enable imagick \
     && docker-php-ext-enable redis \
-    && echo 'memory_limit = 512M' >> /usr/local/etc/php/php.ini \
```
Comment (dsifford) [inline Dockerfile]: Forgot to delete this (dupe of below).

File: Dockerfile [hunk 2]
```diff
@@ -62,7 +61,7 @@ RUN echo "deb http://ftp.debian.org/debian $(sed -n 's/^VERSION=.*(\(.*\)).*/\1/
     # fully qualified domain name
     && echo 'ServerName localhost' > /etc/apache2/conf-available/fqdn.conf \
     # Grab and install wp-cli from remote
-    && curl \
+    && curl --create-dirs \
```
Comment (dsifford) [inline Dockerfile]: In some of the newer base images it appears `/etc/bash_completion.d` isn't available (prob due to bash completion being offloaded or something). So `--create-dirs` fixes that issue.

File: example/README.md [hunk 3]
```diff
@@ -11,35 +11,46 @@ The only thing you need to get started is a `docker-compose.yml` file:
...
+            DB_PASS: root
+            PLUGINS: >-
+                academic-bloggers-toolkit
+                co-authors-plus
+                [WP-API]https://github.com/WP-API/WP-API/archive/master.zip
+
+    db:
+        image: mariadb:10 # or mysql:5.7
+        volumes:
+            - data:/var/lib/mysql
```
Comment (farshidhss) [inline example/README.md]: for mariadb:10 do we keep /var/lib/mysql or need to change it?

**Comment by farshidhss** [comment]
Thank you for making these changes :)

File: example/README.md (hunk 3 above)
Comment (dsifford) [reply example/README.md]: Keep

**Pull request merged**